python/servo/package_commands.py

```diff
@@ -133,6 +137,23 @@ def package(self, release=False, dev=False, android=None, debug=False, debugger=
                 checked.update(checking)
                 need_checked.difference_update(checked)
 
+            print("Adding version to Credits.rtf")
+            vesion_command = [binary_path, '--version']
+            p = subprocess.Popen(vesion_command, stdout=subprocess.PIPE, universal_newlines=True)
+            version, err = p.communicate()
```

Comment:
`err` is misleading IMO (looks like a Go-style function call); let's use `stderr`.